packages/fdb-debugger/src/index.ts

```diff
@@ -43,6 +43,8 @@ export interface ConsoleTrace {
   message: any[];
 }
 
+const FBOS3_EVAL_QUIRK = /^[a-zA-Z]+ \d+\.31\.1\.\d+$/;
```

Comment:
Please add a doc comment describing what the quirk is, why it is necessary and what workaround is required. Maybe even include the relevant issue number from our internal bug tracker. We will have to maintain this workaround for much longer than we will remember what it's for, so it's important to have it all documented here.